### ISSUE TITLE
Inventory Viewer

### ISSUE THREAD

**Issue by Zyrkuh** [runelite/runelite#5049]
**Is your feature request related to a problem? Please describe.**
No, I wouldn't think so - I figure it just hasn't been added yet. I was doing hunter catching orange sallys in desert; I wanted to keep my lunar spellbook open for humidify, but I noticed there's no overlay as there is in the inventory, therefore I couldn't see how many sips my waterskin had left. As well as the rune pouch, but that wasn't really on my mind as I had a thousand casts. Still would be nice to know when it gets low.

**Describe the solution you'd like**
Add it as a feature for not only the normal inventory, but for the inventory viewer.

**Describe alternatives you've considered**
I just turned off inventory viewer and went back to switching between inventory and spellbook.

**Comment by deathbeam** [comment]
I dont think this should be a thing. The inventory viewer simply renders all items in inventory to grid, it has no way to know about runelite overlays in inventory. Also the feature was supposed to be simple with simple use for lunar spells, i dont think it is necessary to extend it further.

**Comment by JaredEzz** [comment]
I would love this. Especially for the fishing barrel, but for other things too
